### PR TITLE
AD/241-logout button on admin header

### PR DESCRIPTION
**Pull request by KonradFak**

### CODE REVIEW

File: src/app/auth/auth.service.ts [hunk 1]
```diff
@@ -29,4 +29,8 @@ export class AuthService {
   remindPassword(email: string) {
     return this.http.post(`${environment.apiUrl}/remind_password`, { email })
   }
+
+  logOut() {
```
Comment (macbem) [inline src/app/auth/auth.service.ts]: It'd be useful to add a return type hint here. It returns an `Observable` of... what? I'd certainly wonder about this if I didn't see the server mocks and I encountered this piece of code in the wild.

File: src/app/common/components/header/header.component.ts [hunk 2]
```diff
@@ -91,6 +93,7 @@ export class HeaderComponent extends HandleSubscription implements OnInit {
   logOut() {
     localStorage.removeItem('adshUser');
     this.store.dispatch(new authActions.SetUser(userInitialState));
+    this.authService.logOut().subscribe();
```
Comment (macbem) [inline src/app/common/components/header/header.component.ts]: Unhandled sub

File: src/app/common/components/header/header.component.ts [hunk 3]
```diff
@@ -36,7 +37,8 @@ export class HeaderComponent extends HandleSubscription implements OnInit {
   constructor(
     private store: Store<AppState>,
     private router: Router,
-    private dialog: MatDialog
+    private dialog: MatDialog,
+    private authService: AuthService
   ) {
     super(null);
```
Comment (macbem) [inline src/app/common/components/header/header.component.ts]: Can't you just call `super` without any parameters?

File: src/app/auth/auth.service.ts (hunk 1 above)
Comment (KonradFak) [reply src/app/auth/auth.service.ts]: Backend developers are about to standardise responses from backend. There would be probably some response code here. I will leave it as it is now but we have open issue to handle such cases.

**Pull request merged**